.travis.yml

```diff
@@ -12,13 +12,13 @@ before_script:
   - composer install --no-interaction --prefer-source --dev
   - ~/.nvm/nvm.sh install v0.6.14
   - ~/.nvm/nvm.sh run v0.6.14
+  - '[ "$TRAVIS_PHP_VERSION" != "7.0" ] || echo "xdebug.overload_var_dump = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini'
```

Comment:
"Before" build failed because overloaded var_dump printed filename and line number.
Setting it to 1 was enough to fix it.